IdIOHandlerWebsocket.pas

```diff
@@ -1526,7 +1526,9 @@ procedure TIdIOHandlerWebsocketSSL.AfterConstruction;
     inherited Write(AStream, ASize, AWriteByteCount) end;
   FWebsocketImpl.FRealIOHandlerI.Inherited_WriteDataToTarget  := function (const ABuffer: TIdBytes; const AOffset, ALength: Integer): Integer begin
     if fSSLSocket <> nil then
-      Result := inherited WriteDataToTarget(ABuffer, AOffset, ALength) end;
+      Result := inherited WriteDataToTarget(ABuffer, AOffset, ALength)
+    else
```

Comment:
Wouldn't it be better to just assign result before condition and get rid of else statement?